### ISSUE TITLE
I tested presence in production and I want to give some feedback

### ISSUE THREAD

**Issue by gabrielhpugliese** [tmeasday/meteor-presence#12]
Hey guys,

I've used your package on production on http://coderstv.com yesterday and I've noticed some strange behaviour in my use case.
I'm using the count of users in determined location of the page. For simplicity, suppose I'm publishing presences for everyone that are on `/`
First, the code:

``` javascript
Template.channel.watchers = function () {
    return Meteor.presences.find({}).fetch().length;
};
```

And the html that prints it:

``` html
<span><i class="icon-user"></i> {{watchers}} watching</span>
```

I had a constant number of watchers of 20. But, sometimes it had shown 30-40 watchers for like 5 seconds and then go back to 20 and stays there for 1-2minutes. And it was a cycle.
I just want to know why would that happen and where of the code is probably doing that behaviour, so I can debug it.

**Comment by tmeasday** [comment]
Is it possible this happened right after a hot code reload?

**Comment by gabrielhpugliese** [comment]
No. It was a live coding session during 1 hour aprox. And it was a cycle for all that long.

**Comment by gabrielhpugliese** [comment]
No ideas ? :(

**Comment by tmeasday** [comment]
A live coding session? What do you mean by this?

**Comment by gabrielhpugliese** [comment]
Oh lol. It's just my product. CodersTV is made for live programming broadcasts :+1:

**Comment by tmeasday** [comment]
Riight so the actual source of the running app wasn't what was being coded on? 

If not then I have no ideas, but the source code of presence is pretty trivial if you want to debug through it.

**Comment by gabrielhpugliese** [comment]
Ok, I will put a lot of debugs in the source code to know what is exactly
happening.
Thanks man :)

Gabriel Pugliese
CodersTV.com
@coderstv

**Comment by gabrielhpugliese** [comment]
Hardest thing ever to debug! :cry:
